### PR TITLE
Add description matcher active hmaconfig

### PR DESCRIPTION
**Pull request by mengyangwang**
Summary
---------
previous PR : https://github.com/facebook/ThreatExchange/pull/516 , https://github.com/facebook/ThreatExchange/pull/544, https://github.com/facebook/ThreatExchange/pull/546
Task : T88063259
     this task is to create ThreatExchange setting page.
     this PR is to add two fields, matcher_active and privacy group description 

Future Actions
---------
 1. Change buttons to icons and adding more tooltips
 2. Add copy text feature for privacy group name(currently has style conflict, might need to rewrite "CopyableTextField" module later)
 3. use matcher_active to disable matching  
 4. update description to add copydays link

Test Plan
---------
1. black hmalib
2. mypy hmalib
3. python -m py.test to pass all tests
4. make docker
5. make upload_docker 
6. terraform apply
7. test get/update/sync/delete api for matcher_active and description

https://user-images.githubusercontent.com/81996660/117205046-148bd800-adbf-11eb-9477-794e540d946c.mov

### CODE REVIEW

File: hasher-matcher-actioner/hmalib/common/threatexchange_config.py [hunk 1]
```diff
@@ -8,12 +8,14 @@
...
 from hmalib.common import config as hmaconfig
-from hmalib.lambdas.fetcher import ThreatExchangeConfig
+from hmalib.common.config import HMAConfig
+from hmalib.common.fetcher_models import ThreatExchangeConfig
 from botocore.exceptions import ClientError
 
 logger = get_logger(__name__)
 FETCHER_ACTIVE = True
 WRITE_BACK = True
+MATCHER_ACTIVE = True
```
Comment (jeberl) [inline hasher-matcher-actioner/hmalib/common/threatexchange_config.py:18]: Can we make these defaults? eg. MATCHER_ACTIVE_DEFAULT

**Review by jeberl — APPROVED** [review]
Looks great!

**Review by jeberl — APPROVED** [review]
Looks great!

File: hasher-matcher-actioner/hmalib/common/threatexchange_config.py (hunk 1 above)
Comment (mengyangwang) [reply hasher-matcher-actioner/hmalib/common/threatexchange_config.py:18]: sure will do

**Pull request merged**